src/NHibernate.Test/NHSpecificTest/NH1007/Mappings.hbm.xml

```diff
@@ -6,18 +6,15 @@
 		<property name="Name" />
 	</class>
 
-	<!-- 
-		Mapping to demonstrate that the 'column' attribute takes precedence over a 'column' element - and, for consistency,
-		the same behaviour applies to the generator attribute/element
```

Comment:
Wrong for `column`: that was the element which was taking precedence.
([NH-1007](https://nhibernate.jira.com/browse/NH-1007) is about adding the `generator` attribute as an alternate to specifying the element, when the user just needs to specify the generator name.)